languages/Payment/mn.ini

```
@@ -0,0 +1,40 @@
+Accept = "Зөвшөөрөх"
+breakdown_title = "Төлбөрийн баримт"
+Cancel = "Цуцлах"
+Contact Information = "Холбоо барих мэдээлэл"
+Date = "Огноо"
+Details = "Дэлгэрэнгүй"
+download_receipt_pdf = "Төлбөрийн баримт татах (PDF)"
+email_hello = "Сайн уу!"
+email_receipt_footer = "Энэ бол номын сангийн автомат мессеж юм. Энэ зурваст хариу бичих хэрэггүй. Хэрэв танд асуулт байвал номын сантай холбогдоно уу."
+email_receipt_message = "Таны төлбөрийн баримтыг PDF файл хэлбэрээр хавсаргав."
+error_fines_changed = "Торгуулийн жагсаалтыг өмнөх хуудсыг ачаалсны дараа шинэчилэгдэнэ."
+error_payment_request_failed = "Төлбөрийн хүсэлт амжилтгүй боллоо"
+Excluding VAT = "НӨАТ ороогүй"
+Fee = "Төлбөр"
+fines_contain_nonpayable_fees = "Танд онлайнаар төлөх боломжгүй хураамж байна. Номын сангийн харилцагчийн үйлчилгээтэй холбогдоно уу."
+Identifier = "Тодорхойлогч"
+in_progress_warning = "Өмнө нь төлбөрийг %%startDateTime%%-д эхлүүлсэн. Зөвхөн өмнөх төлбөр тасалдсан тохиолдолд шинэ төлбөрийг эхлүүлнэ үү."
+Including VAT = "НӨАТ орсон"
+Last Paid = "Сүүлийн төлсөн"
+minimum_payment = "Төлбөрийн доод хэмжээ"
+nonpayable_fee = "Энэ төлбөрийг онлайнаар төлөх боломжгүй"
+payable_online = "Онлайнаар төлөх"
+Payment Canceled = "Цуцалсан төлбөр"
+Payment Successful = "Төлбөр амжилттай"
+Processing Payment = "Төлбөрийг хийгдэж байна"
+receipt_no_details_placeholder = "-"
+Recipient = "Төлбөрийн хүлээн авагч"
+registration_failed = "Төлбөрийг таны банкны данснаас авсан боловч таны номын сангийн дансны бүртгэл сунгагдаагүй. Бүртгэл дуустал төлбөртэй харагдах болно."
+remaining_balance_after_payment = "Төлбөрийн дараа үлдсэн үлдэгдэл"
+selected_to_pay = "Сонгосон төлбөр хийх"
+Service Fee = "Үйлчилгээний хураамж"
+start_payment = "Онлайнаар төлөх %%amount%%"
+terms_heading = "Төлбөрийг үргэлжлүүлэх нөхцөлийг хүлээн зөвшөөр"
+terms_paytrail_html = "Дараах мэдээллийг төлбөрийн зуучлагч Paytrail-д шилжүүлэх болно: <ul class='list-bullets'><li>Таны нэр</li><li>Таны имэйл хаяг</li><li>Таны номын сангийн картын дугаар</li><li>Таны төлөх төлбөрийн талаарх мэдээлэл</li></ul> Мөн Paytrail хуудаснаас үзнэ үү. <a href='https://www.paytrail.com/en/consumer/information-about-paying' target='_blank' data-lightbox-ignore>Төлбөрийн талаарх мэдээлэл</a>."
+terms_stripe_html = "Дараах мэдээллийг төлбөрийн зуучлагч Stripe-д дамжуулах болно: <ul class='list-bullets'><li>Таны нэр</li><li>Таны цахим шуудангийн хаяг</li><li>Таны номын сангийн картын дугаар</li><li>Таны төлөх төлбөрийн талаарх мэдээлэл</li></ul> Мөн Stripe-ийн үзнэ үү <a href='https://stripe.com/en-fi/privacy' target='_blank' data-lightbox-ignore>Нууцлалын бодлого</a>"
```

Looks like the final punctuation got dropped here in the last edit. I'm putting it back via suggestion here:

```suggestion
terms_stripe_html = "Дараах мэдээллийг төлбөрийн зуучлагч Stripe-д дамжуулах болно: <ul class='list-bullets'><li>Таны нэр</li><li>Таны цахим шуудангийн хаяг</li><li>Таны номын сангийн картын дугаар</li><li>Таны төлөх төлбөрийн талаарх мэдээлэл</li></ul> Мөн Stripe-ийн үзнэ үү <a href='https://stripe.com/en-fi/privacy' target='_blank' data-lightbox-ignore>Нууцлалын бодлого</a>."
```